docs/cli/teams/shared.md

```diff
@@ -3,7 +3,7 @@ title: Share single environment
 description: Learn the recommended workflow for multiple team members sharing a single Amplify environment.
 ---
 
-You have two independent environments (`main` & `dev`) in the cloud and have corresponding Git branches with your Amplify backend infrastructure code on Git. Suppose all team members want to work on the same Amplify project and push backend related changes to the same `dev` environment to test their changes. Each team member would run the following:
+You have two independent environments (main & dev) in the cloud and have corresponding git branches with your amplify backend infrastructure code on Git. Suppose all team members want to work on the same Amplify project and push backend related changes to the same dev environment to test their changes. Each team member would run the following:
```

Comment:
```suggestion
You have two independent environments (`main` & `dev`) in the cloud and have corresponding Git branches with your Amplify backend infrastructure code on Git. Suppose all team members want to work on the same Amplify project and push backend related changes to the same `dev` environment to test their changes. Each team member would run the following:
```

Reply:
Git is correct (as opposed to git), as is Amplify vs. amplify.  Let's keep `main` and `dev` as opposed to main and dev as well.